fastlane_bot/bot.py

```diff
@@ -312,55 +272,36 @@ def _run(
         Parameters
         ----------
         flashloan_tokens: List[str]
-            The tokens to flashloan.
+            The flashloan tokens
         CCm: CPCContainer
-            The container.
+            The complete market data container
```

Comment:
I am not sure of the meaning of "complete" here. Is it the entire market at this stage, or only the curves for the optimization?